docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -0,0 +1,239 @@
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hbase-rowset-vssb-opt]]
+== HBASE_ROWSET_VSSB_OPT
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Allows INSERT, UPDATE, and DELETE (IUD) operations to be performed as an HBase batch `put` operation. 
+| *Values*                    |
+*'ON'*: Perform IUD operations as an HBase batch `put` operation. +
+*'OFF'*: Do not perform IUD operations as an HBase batch `put` operation. +
+ +
+The default value is *'ON'*.
+| *Usage*                     | When IUD operation involves multiple tuples, then the Trafodion Optimizer evaluates whether these operations
+can be done in a batch manner at the HBase level thereby reducing the network interactions between the client applications and the HBase Region Servers. +
+ +
+If possible, then the query plan involves VSBB operators. The Virtual Sequential Block Buffer(VSBB) name is retained in Trafodion though it is unrelated to HBase.
+| *Production Usage*          | Yes.
+| *Impact*                    | IUD operations can become slower if this CQD is set to 'OFF'.
+| *Level*                     | Query.
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hbase-rowset-vssb-size]]
+== HBASE_ROWSET_VSSB_SIZE
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Determines the maximum number of rows in a batch `put` operation to HBase. 
+| *Values*                    |
+Numeric value.
+ +
+The default value is *'1024'*.
+| *Usage*                     | The Trafodion execution engine already adjusts the number of rows in a batch depending upon how fast
+the queue to IUD (INSERT,UPDATE,DELETE) operator is filled up in the data flow architecture of Trafodion. +
+ +
+You can adjust the maximum size to suit your application needs and thus tune it to perform optimally.
+| *Production Usage*          | Yes. You can disable this feature by setting the HBASE_ROWSET_VSSB_OPT CQD to 'OFF'.
+| *Impact*                    | The performance of your application may be affected by setting this CQD too low.
+| *Level*                     | Query.
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hbase-small-scanner]]
+== HBASE_SMALL_SCANNER
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Enables Trafodion to leverage the HBase small scanner optimization. This optimization reduces I/O usage with up to 66% 
+and enables non-blocking reads for higher concurrency support. When a scan is known to require less than a HBASE BLOCK SIZE (default is 64K),
```

Comment:
In the phrase, "reduces I/O usage with up to 66%" suggest striking the word "with"